dlang.org.ddoc

```diff
@@ -211,6 +211,12 @@ _=
 PAGE_TOOLS=
 $(DIVID tools,
 	$(SPANC tip,
+		<a href="https://issues.dlang.org/enter_bug.cgi?bug_file_loc=http%3A%2F%2Fdlang.org/$(SELF_PATH)&bug_severity=enhancement&component=$(BUGZILLA_COMPONENT)&op_sys=All&priority=P3&product=D&rep_platform=All&short_desc=%5B$(TITLE)%5D&version=D2" class="button">Report a bug</a>
```

Comment:
Eh, done, but I don't see the point frankly. It's longer and carries no semantic meaning (there is no meaningful alternative value for HTMLTAG3 one may want to use)..